app/components/location-map.js

```diff
@@ -3,6 +3,10 @@ import Component from '@ember/component';
 
 export default Component.extend({
   classNames: ['map-container'],
+  attributeBindings: ['aria-hidden:ariaHidden', 'tabindex'],
```

Comment:
For the sake of the tutorial I'm wondering if I should just add `aria-hidden` and `tabindex` as attributes to the component invocation.  I might be easier to explain then introducing tutorial users to attributeBindings.

ex.  {{location-map aria-hidden="true" tabindex="-1"}}

What do you think @jenweber